src/Plugin/Field/FieldFormatter/AudioPlayerHTML5.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\media_entity_audio\Plugin\Field\FieldFormatter\AudioPlayerHTML5.
+ */
+
+namespace Drupal\media_entity_audio\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Entity\EntityStorageInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Routing\UrlGeneratorInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\Core\Url;
+use Drupal\Core\Utility\LinkGeneratorInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Cache\Cache;
+
+//use Drupal\file\Plugin\Field\FieldFormatter\GenericFileFormatter;
+
+/**
+ * Plugin implementation of the 'Audio Player (HTML5)' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "audio_player_html5",
+ *   label = @Translation("Audio Player (HTML5)"),
+ *   field_types = {
+ *     "file"
+ *   }
+ * )
+ */
+class AudioPlayerHTML5 extends AudioPlayerBase implements ContainerFactoryPluginInterface{
+
+
+/**
+   * Constructs an ImageFormatter object.
+   *
+   * @param string $plugin_id
+   *   The plugin_id for the formatter.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Field\FieldDefinitionInterface $field_definition
+   *   The definition of the field to which the formatter is associated.
+   * @param array $settings
+   *   The formatter settings.
+   * @param string $label
+   *   The formatter label display setting.
+   * @param string $view_mode
+   *   The view mode.
+   * @param array $third_party_settings
+   *   Any third party settings settings.
+   */
+  public function __construct($plugin_id, $plugin_definition, FieldDefinitionInterface $field_definition, array $settings, $label, $view_mode, array $third_party_settings) {
+    parent::__construct($plugin_id, $plugin_definition, $field_definition, $settings, $label, $view_mode, $third_party_settings);
```

Comment:
Is there any special reason why it implements it? We usually implement this if we want to inject a service from the container into the plugin. It seems that this is not the case here.